pipeline/dags/task_factories.py

```diff
@@ -89,10 +89,10 @@ def fn(filename: str):
 def gcloud_upload_dir(layer_id: str, layer_variable: str, directory: str):
     return BashOperator(
         task_id='gcloud_upload',
-        bash_command='gcloud auth activate-service-account --key-file $KEY_FILE && gsutil -q -m cp -r $UPLOAD_DIR/* $BUCKET',
+        bash_command='gcloud auth activate-service-account --key-file $KEY_FILE && gsutil -m -h "Cache-Control:no-cache" rsync -d -r $UPLOAD_DIR $BUCKET',
```

Comment:
Just wondering, why the `no-cache` parameter=

Reply:
You are right. I copied the command from somewhere, but in this case caching makes sense.